test/fixtures/install_simple/policy-library/policies/templates/gcp_sql_public_ip_v1.yaml

```diff
@@ -41,7 +41,8 @@ spec:
          # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
          # See the License for the specific language governing permissions and
          # limitations under the License.
-         #
+         package templates.gcp.GCPSQLPublicIpConstraintV1
+         import data.validator.gcp.lib as lib
```

Comment:
It looks like these lines may be duplicated in this file?  OOC, should this file be included in this PR?

Reply:
Looks like these file was updated by one of the `make` commands, probably docker generate. Anyway, I have removed this change from this PR.